mfr/core/utils.py

```diff
@@ -98,44 +98,46 @@ def make_renderer(name, metadata, file_path, url, assets_url, export_url):
             }
         )
 
-def get_renderer_name(name):
+
+def get_renderer_name(name: str) -> str:
     """ Return the name of the renderer used for a certain file extension.
 
     :param str name: The name of the extension to get the renderer name for. (.jpg, .docx, etc)
 
     :rtype : `str`
     """
 
-    # This can give back empty tuples
-    try:
-        entry_attrs = pkg_resources.iter_entry_points(group='mfr.renderers', name=name)
-
-        # ep.attrs is a tuple of attributes. There should only ever be one or `None`.
-        # None case occurs when trying to render an unsupported file type
-        # entry_attrs is an iterable object, so we turn into a list to index it
-        return list(entry_attrs)[0].attrs[0]
+    # `ep_iterator` is an iterable object. Must convert it to a `list` for access.
+    # `list()` can only be called once because the iterator moves to the end after conversion.
+    ep_iterator = pkg_resources.iter_entry_points(group='mfr.renderers', name=name.lower())
```

Comment:
For both exporter and renderer, the primary functional change is `name=name.lower()`.